packages/validator/docs/page.mdx

```diff
@@ -18,10 +18,13 @@ These are the breaking changes that came with the validator V2:
 
 - The `ruler()` has been removed and replaced with `field()`
 - The `.type()` function is removed and `field()` replaces it. E.g.: `validator.field('email')`
+- The `.present()` function is deprecated and `present()` replaces it.
```

Comment:
A rework on this doc page is coming in the future as well

Reply:
In the docs rework we will add a page specific to conditionally running checks using `when`, `and`, `run` and `else`